### DISCUSSION TITLE
Remote Start with Climate Doc Update

### DISCUSSION THREAD

**Discussion by jdblue225** [G-Two/homeassistant-subaru#73] [Q&A]
Is there an update to the documentation for remote start with climate presets? It seems subaru.remote_start service was removed in one of the updates and migrated to HA buttons. I don't understand how to send the climate preset data anymore seeing as before you could just add it to the service request. Thanks!

**Comment by stboch** [comment]
You should see a select.(car name)_climate_preset this is what will be send when the remote start button is pressed. Nice for you can set an automation to change your preset based on weather and then all you do is trigger the button.
